README.md

```diff
@@ -1,63 +1,99 @@
 - [ManySecured Docs: D3](https://specs.manysecured.net/d3)
 - [ManySecured Docs: D3 Claims](https://specs.manysecured.net/d3/D3%20claims)
 - [ManySecured Docs: D3 Behaviours](https://specs.manysecured.net/d3/D3%20behaviourss)
 
+## D3 Architecture Summary
+
+![Relationship Graph for the D3 Claim Types](./examples/D3-claim-dep-graph.png)
+
+### D3 Claim Type Summaries
+
+#### Device TYPE assertions
+
+Device types are the core description of an instance of a given device. The type assertion encapsulates the device's details and refers (by GUID reference) to the behaviours, vulnerabilities, and firmware associated with that device. Type assertions operate an inheritance model, where a type assertion can inherit the properties from parent types and then overload/add properties unique to that device type instance.
+
+#### Device BEHAVIOUR assertions
+
+Device behaviour claims detail the activity the device should and should not exhibit on the network. A behaviour claim is a collection of multiple network activity rules which define the network activity parameters the device should comply with. Behaviour rules are one of two types, expected and malicious. Expected rules detail the activity that a device is expected to conform to under normal operation. Malicious rules define network activity the device should categorically not exhibit (these can usually be inherited from the master behaviour definition for a pre-defined list of known malicious activity).
+
+On a ManySecured enabled router, the router can use the behaviour claim to check the connected device behaves according to its type and identify unexpected behaviour (behaviour that does not match any expected rule) and suspicious behaviour (behaviour that matches a malicious behaviour rule).
+
+Behaviour claims comprise a set of rules and can also elect to inherit the rules from other behaviours into a more comprehensive set of rules, e.g. The behaviour claim for a 3-in-one printer might inherit the behaviour claims of a printer, scanner, and copier.
+
+#### Device FIRMWARE assertions
+
+Firmware assertions are used to describe the firmware versions a device can have, provide links to the spec, and be used to match against known vulnerabilities for that firmware.
+
+#### Device VULNERABILITY assertions
+
+Vulnerability assertions allow claims to be made about the vulnerabilities associated with the device. Vulnerability claims are generally auto-populated from the NIST and CVE vulnerability databases but can also be manually added.
+
 ## How to add a new D3 claim
 
 For contributing changes to code, please see [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-1. Start a new branch.
-2. Within the `manufacturers folder` folder, create the claim files for the manufacturers products. If you organisae your files into subdirectories they will be recuresed into automatically at compile time.
-3. Add a YAML file with the details for your D3 spec. See
+1. **Start a new branch**.
```

Comment:
Most contributors won't be able to start a new branch, since they don't have permission, however they can create their own fork and start branches there.

```suggestion
1. [Fork this repo](https://github.com/TechWorksHub/ManySecured-D3DB/fork), and **start a new branch**.
```